public/apps/account/account-nav-button.tsx

```diff
@@ -72,9 +76,11 @@ export function AccountNavButton(props: {
   React.useEffect(() => {
     const fetchData = async () => {
       try {
-        setIsMultiTenancyEnabled(
-          (await getDashboardsInfo(props.coreStart.http)).multitenancy_enabled
+        // defaults to true if for some reason the multitenancy_enabled flag is undefined. This should ideally never happen.
+        const { multitenancy_enabled: isEnabled = true } = await getDashboardsInfo(
+          props.coreStart.http
         );
+        setIsMultiTenancyEnabled(isEnabled);
       } catch (e) {
         // TODO: switch to better error display.
         console.error(e);
```

Comment:
we should have default behavior instead of print error into browser console which will notice nothing to customer and not actonable